### ISSUE TITLE
[Bug]: Program crashes if it encounters a video with disabled comments

### ISSUE THREAD

**Issue by Trainmaster2** [ThioJoe/YT-Spammer-Purge#346]
### Duplicate Issues

- [X] There are **no existing posts relating to my problem**
- [ ] There are **existing posts relating to my problem, but the solution given, doesn't work for me.**

### What happened?

I was preparing myself to lose faith in humanity by running a "Scan specific video" on the Baby Shark video when the program crashed. After a couple more attempts I noticed the error mentioned "commentCount". So I went and checked the video and, lo and behold, comments were disabled.

### Release version

2.7.4 and 2.8.1

### Steps to reproduce

Run a scan on a specific video that has comments disabled.

### What platform are you seeing this problem on?

Windows

### Relevant log output

```shell
Traceback (most recent call last):
  File "D:\Users\Joe\Documents\GitHub ThioJoe\YouTube-Spammer-Purge\YouTubeSpammerPurge.py", line 3726, in <module>
  File "D:\Users\Joe\Documents\GitHub ThioJoe\YouTube-Spammer-Purge\YouTubeSpammerPurge.py", line 2928, in main
  File "D:\Users\Joe\Documents\GitHub ThioJoe\YouTube-Spammer-Purge\YouTubeSpammerPurge.py", line 1163, in validate_video_id
KeyError: 'commentCount'
------------------------------------------------
Unknown Error - Code: X-4 occurred. This is probably my fault,
please a bug report on the GitHub issues page, and include the above error info.
Short Link: TJoe.io/bug-report
```

**Comment by ThioJoe** [comment]
Ah ok I'll add a bit of extra handling to better notify the reason for this error

**Comment by ThioJoe** [comment]
Should be fixed now, and gives comments being disabled as a likely reason when it occurs, will be included when I make the patch release 2.8.2 soon.
